be/src/exec/hdfs_scanner_orc.cpp

```diff
@@ -301,6 +301,7 @@ bool OrcRowReaderFilter::filterOnPickStringDictionary(
                                                  filter_ptr);
         if (!status.ok()) {
             LOG(WARNING) << "eval conjuncts fails: " << status.message();
+            _dict_filter_eval_cache.erase(slot_id);
```

Comment:
in what case this status is not ok?

Reply:
some conjunct will eval failed, will return Status::InternalError

Reply:
some predicates like `split(str, '_')[2] = 'xxx'` could fail.